css-positioning/absolute.css

```diff
@@ -0,0 +1,41 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  left: 230px;
```

Comment:
Oh no, this is a **_massive_** movement. Remember what I said when introducing the exercise, you should try to use small movements from the side closest to where you want the content to be. I would suggest taking another look at this piece of the instructions. Development requires **_very careful reading_**.
![image](https://user-images.githubusercontent.com/32880187/151609422-0c46588a-7c7b-41da-888e-ff667feef317.png)